marklogic-data-hub/src/test/java/com/marklogic/hub/provenance/MigrateProvenanceTest.java

```diff
@@ -0,0 +1,18 @@
+package com.marklogic.hub.provenance;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MigrateProvenanceTest  extends ProvenanceTest {
+    @Test
+    void migrateProvenanceRecords() {
+        installProjectAndRunFlow();
+        ProvenanceManager provenanceManager = new ProvenanceManager(getHubClient());
+        assertEquals(2, getJobsDocCount(provenanceCollection),
```

Comment:
I think it'd be helpful to add an assertion after the migration on whether or not the provenance count in the jobs database is zero or 2, along with an assertion stating why the docs are either deleted or not deleted.

Reply:
Added another assertion in this test with some explaination.